src/components/dashboard/Claim/Phase0.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import React from 'react'
+import { View } from 'react-native'
+import numeral from 'numeral'
+import Config from '../../../config/config'
+
+import { weiToGd } from '../../../lib/wallet/utils'
+import { WrapperClaim } from '../../common'
+import BigGoodDollar from '../../common/view/BigGoodDollar'
+import Text from '../../common/view/Text'
+
+import Section from '../../common/layout/Section'
+import ClaimButton from '../ClaimButton'
+
+const Claimphase0 = ({ styles, isCitizen, entitlement, state, handleClaim, faceRecognition }) => {
+  return (
+    <WrapperClaim>
+      <Section style={styles.mainContainer}>
+        <Section.Stack style={styles.mainText}>
+          <View style={styles.mainTextBorder}>
+            <Section.Text>
+              <Section.Text color="surface" fontFamily="slab" fontWeight="bold" fontSize={40}>
+                {entitlement ? `Claim Your\nDaily Share` : `Just a Few More\nHours To Go...`}
+              </Section.Text>
+            </Section.Text>
+          </View>
+          {entitlement > 0 ? (
+            <Section.Row alignItems="center" justifyContent="center" style={[styles.row, styles.subMainText]}>
+              <View style={styles.amountBlock}>
+                <Text color="#0C263D" fontSize={55} fontWeight="bold" fontFamily="Roboto">
+                  <BigGoodDollar
+                    reverse={true}
+                    number={entitlement}
+                    formatter={weiToGd}
+                    fontFamily="Roboto"
+                    bigNumberProps={{
+                      fontFamily: 'Roboto',
+                      fontSize: 50,
+                      color: 'black',
+                      fontWeight: 'bold',
+                      lineHeight: 36,
+                    }}
+                    bigNumberUnitProps={{
+                      fontFamily: 'Roboto',
+                      fontSize: 45,
+                      color: 'black',
+                      fontWeight: 'medium',
+                      lineHeight: 20,
+                    }}
+                  />
+                </Text>
+              </View>
+            </Section.Row>
+          ) : null}
+          <View>
+            <Section.Text style={styles.secondTextBlock}>
+              <Section.Text color="surface" fontFamily="slab" fontSize={18}>
+                {`GoodDollar is the world’s first experiment\nto create a framework to generate\nUBI on a global scale.\n`}
+                <Text
+                  color="surface"
+                  style={styles.learnMoreLink}
+                  textDecorationLine="underline"
+                  fontSize={18}
+                  fontWeight="bold"
+                  fontFamily="slab"
+                  onPress={() => window.open(Config.urlToLearnMore, '_blank')}
```

Comment:
Won't work on native, please, use https://reactnative.dev/docs/linking abstraction

Reply:
and wrap to callback pls or move outside component as it's props-independent